rclcpp/include/rclcpp/create_publisher.hpp

```diff
@@ -92,7 +92,7 @@ template<
   typename NodeT>
 std::shared_ptr<PublisherT>
 create_publisher(
-  NodeT & node,
+  NodeT && node,
```

Comment:
could you add a test case showing that https://github.com/ros2/rclcpp/issues/1876 is fixed after this?

Reply:
@ivanpauno totally makes sense :+1: